src/HttpTransport.cpp

```diff
@@ -10,29 +9,42 @@ using namespace apache::thrift;
 
 namespace cppkin
 {
+
+//#ifdef THRIFT_FOUND
+//    constexpr EncodingTypes::Enumeration _EncodingType = EncodingTypes::Thrift;
+//    constexpr const char* _ContentType = "Content-Type: application/x-thrift";
+//#else
+//    constexpr EncodingTypes::Enumeration _EncodingType = EncodingTypes::Json;
+//    constexpr const char* _ContentType = "Content-Type: application/json";
+//#endif
+
+    constexpr EncodingTypes::Enumeration _EncodingType = EncodingTypes::Json;
```

Comment:
constexpr is not supported by vc++12, so this will cause an issue with our windows version.
also, why not just const? there is not real logic needed to be commenced at compile time, just initialization.

also, why the aliasing with the Enum value? why not using it directly in the template.